R/pull.R

```diff
@@ -191,14 +191,8 @@ append_predictions <- function(collection, predictions, split, control, .config
 }
 
 append_extracts <- function(collection, workflow, grid, split, ctrl, .config = NULL) {
-  extracts <-
-    grid %>%
-    dplyr::bind_cols(labels(split)) %>%
-    mutate(
-      .extracts = list(
-        extract_details(workflow, ctrl$extract)
-      )
-    )
+  extracts <- dplyr::bind_cols(grid, labels(split))
+  extracts$.extracts <- list(extract_details(workflow, ctrl$extract))
```

Comment:
`mutate()` wraps the raised errors with information that, when collapsed by tune's machinery, obscures the condition messages raised from the extractor functions themselves.